src/sass/variables.scss

```diff
@@ -0,0 +1,12 @@
+@charset "UTF-8";
```

Comment:
Is there some way we can differentiate this from `base/b-variables`? If I'm understanding this correctly, all the `@import`s here are for variables of all kinds, but I think it may get confusing to have a `variables.scss` and a `base/_b-variables.scss`.